api/goldens/aptos_api__tests__transactions_test__test_get_transactions_output_user_transaction_with_entry_function_payload.json

```diff
@@ -237,7 +237,7 @@
     ],
     "sender": "0xa550c18",
     "sequence_number": "0",
-    "max_gas_amount": "20000",
+    "max_gas_amount": "100000",
     "gas_unit_price": "0",
```

Comment:
Somewhat concerning that creating account costs 0.001 APT. We can revisit this later.